### PR TITLE
Fix gem dependencies

### PR DESCRIPTION
**Pull request by robwise**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,7 +94,6 @@ We're definitely not doing that. With react_on_rails, webpack is mainly generati
   
   ```ruby
   gem "react_on_rails"
-  gem "therubyracer"
   ```
```
Comment (justin808) [inline README.md:97]: add line under indicating to add this for server rendering, if desired

File: docs/generator_testing_script.md [hunk 2]
```diff
@@ -15,7 +15,6 @@ Edit the Gemfile, adding these 2 lines:
 
 ```ruby
 gem 'react_on_rails', path: '../react_on_rails'
-gem 'therubyracer'
```
Comment (justin808) [inline docs/generator_testing_script.md:18]: this should stay for testing

File: README.md (hunk 1 above)
Comment (justin808) [inline README.md:97]: I realize that we add it with the generator, but it's probably important enough to mention it.

File: docs/generator_testing_script.md (hunk 2 above)
Comment (robwise) [inline docs/generator_testing_script.md:18]: @justin808 It's in the generator now though?

File: README.md (hunk 1 above)
Comment (robwise) [inline README.md:97]: @justin808 good call, I mention it in the server rendering section in 3e4229b

**Comment by robwise** [comment]
@justin808 this should be good to go pending a final review

File: docs/generator_testing_script.md (hunk 2 above)
Comment (justin808) [inline docs/generator_testing_script.md:18]: OK

**Comment by justin808** [comment]
If this is well tested, let me know and let's merge. Looks good.

**Pull request merged**